### DISCUSSION TITLE
Update to zwavejs/zwave-js-ui.

### DISCUSSION THREAD

**Discussion by pvhpsb** [zwave-js/zwave-js-ui#4051] [Q&A]
I use Domoticz and now have zwavejs/zwavejs2mqtt and want to update to zwavejs/zwave-js-ui.
How can I do that without losing data.

**Comment by robertsLando** [comment]
@pvhpsb are you using docker?

**Reply by pvhpsb** [reply]
Yes on a Virtual Ubuntu

**Reply by robertsLando** [reply]
If so all the useful data are stored on store volume/dir (based on if you are mapping store to a docker volume or a local dir). What you have to do is to simply change the tag name to the new one and keep the other settings unchanged. Show me your docker-compose file

**Reply by pvhpsb** [reply]
I cant find a docker-compose.yml

**Reply by robertsLando** [reply]
how did you started the container? Did you use a command?

**Reply by pvhpsb** [reply]
long time ago:, i remember      docker run -d --name zwavejs2mqtt --restart unless-stopped -v /opt/zwavejs2mqtt/store:/usr/src/app/store -p 8091:8091 -p 3000:3000 --device=/dev/ttyACM0  zwavejs/zwavejs2mqtt:latest

**Reply by pvhpsb** [reply]
but i'm not sure

**Reply by robertsLando** [reply]
Do you have something on  /opt/zwavejs2mqtt/store? Because if so that's the important part. 

To be sure you can do a backup of store from Store tab UI and an NVM backup as well (even if NVM is not really needed). Once done simply stop the container, run  same command by using the new tag and everything should be there as before. If you don't see anything you can restore the backup of the store you previously did from Store Tab.

Anyway I suggest you to use a docker-compose file instead so you have a reference of the service you are running and all the options passed
